src/pages/home/Home.jsx

```diff
@@ -0,0 +1,67 @@
+import { useEffect, useState } from "react";
+import { instance } from "../../api/api";
+import {
+  Container,
+  Banner,
+  Intro,
+  List,
+  Logo,
+  Item,
+  Thum,
+  Info,
+  Name,
+  Price,
+} from "./styles";
+
+const Items = ({ props }) => {
+  return (
+    <Item>
+      <Thum src={props.thumbnail} alt="상품이미지" />
+      <Info>
+        <Name>{props.title}</Name>
+        <Price>{props.price.toLocaleString()}</Price>
+      </Info>
+    </Item>
+  );
+};
+
+const Home = () => {
+  const [data, setData] = useState([]);
```

Comment:
```suggestion
  const [products, setProducts] = useState([]);
```

이런식으로 네이밍을 하면 좀 더 가독성을 높일 수 있을 것 같아요!